src/config/dynamic-theme-fixes.config

```diff
@@ -1443,6 +1443,15 @@ CSS
 
 ================================
 
+dw.com
+
+CSS
+div#bodyMover {
```

Comment:
`!important`

Reply:
Is `div` needed?

Reply:
You are right. In my previous testing adding div avoids using `!important` but it's... unstable. Sorry!